composer.json

```diff
@@ -55,11 +58,14 @@
         "sonata-project/core-bundle": "^3.14",
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
-        "symfony-cmf/testing": "^2.1.12",
+        "symfony-cmf/testing": "^3@dev",
```

Comment:
i just tagged 3.0.0 now. can you please change this constraint to `^3.0` without the `@dev`?